README.md

```diff
@@ -37,10 +37,11 @@ Or install it yourself as:
 
 ```ruby
 YourApp::Application.configure do
-  config.npmdc.path   = "/path/to/your/frontend/code/dir" # `Rails.root` by default
-  config.npmdc.format = "doc"                             # `short`, `doc`, `progress`. `short` by default
-  config.npmdc.color  = false                             # `true` by default
-  config.npmdc.types  = ["dependencies"]                  # `["dependencies", "devDependencies"]` by default
+  config.npmdc.path              = "/path/to/your/frontend/code/dir" # `Rails.root` by default
+  config.npmdc.format            = "doc"                             # `short`, `doc`, `progress`. `short` by default
+  config.npmdc.color             = false                             # `true` by default
+  config.npmdc.types             = ["dependencies"]                  # `["dependencies", "devDependencies"]` by default
+  config.npmdc.development_only  = true                              # true by default
```

Comment:
I'd like to have a more flexible option, say, `config.npmdc.environments = [:development]`, 'cause sometimes it make sense to use it in `test` too (when you have a lot of acceptance/features specs depending on assets).

And there can also be _custom_ environments too.